### DISCUSSION TITLE
Fixed scale for brand colors

### DISCUSSION THREAD

**Discussion by wilson-webdev** [chakra-ui/chakra-ui#4978] [Q&A] [answered]
Let's say that I am implementing Chakra UI for my company's platform and the main branding color is `#12d4af`. This would be the `.500` value on Chakra's scale. When switching between light and dark mode, components select a relevant color from this scale (for example, the `Button` component will set the `bg` to `.200`). Also, upon `_hover`, the `Button` component will select `.300` as the `bg` color for dark mode with the `filled` variant. 

This is great for most colors as it's really powerful to have auto coloring for good contrast etc. However, the branding color needs to be fixed across all component (e.g. `.500` everywhere) for both light and dark mode, is there a way of doing this but still maintaining effects such as the hover state being slightly darker?

**Comment by primos63** [comment, marked as answer]
If you're looking for a special setting to do such, then I'm afraid not.

But, if you plan on having the brand color defined with levels, 50 - 900,
```javascript
const colors = {
  brand: {
    50: "#e7fdf9",
    100: "#b8f9ed",
    200: "#89f5e1",
    300: "#5af1d4",
    400: "#2bedc8",
    500: "#12d4af",
    600: "#0ea588",
    700: "#0a7661",
    800: "#06473a",
    900: "#021813"
  }
}
```
then you can achieve your goal. You'll need to extend your theme so that the component color is configured as you wish it to be. That will require looking at the theme source for the components you wish to use and changing the color parts you desire to have the branding.
Here's a [CodeSandbox example](https://codesandbox.io/s/musing-hamilton-g15h3?file=/src/App.js). It's not complete. It's meant as a demonstration of what can be done.

If you have any other questions, let me know.

**Reply by wilson-webdev** [reply]
Thank you, @primos63! It's good that you can merge Chakra's out-of-the-box variant definitions with your own overrides which avoids a lot of boilerplate. Your CodeSandbox is a massive help, thank you!
